### ISSUE TITLE
Confusing `short` shortcut

### ISSUE THREAD

**Issue by bambinoua** [isar/isar#949]
Actually (you know), the `short` is 2 byte integer [−32767, 32767] so maybe it is good to "decrease" its size. I understand that maybe you would like to emulate a half of `int` (similar to C-like byte,short,int,long) but having `short` as 2 byte integer and `int` for other integers (4 and 8) avoids the confusing.

**Comment by leisim** [comment]
I agree. What name do you suggest for 4 byte integers? `int` is already taken by Dart for 8 byte integers ☹️

**Comment by bambinoua** [comment]
As I understand you need new names for distinction? So here it is the solution:

```dart
typedef byte = int;
typedef short = int;
typedef int32 = int;
typedef int64 = int;
typedef long = int;
```

**Comment by leisim** [comment]
Numbers are not recognised by syntax highlighting 😭

**Comment by bambinoua** [comment]
Shit happens :)
```dart
typedef byte = int;     // 1 byte
typedef short = int;    // 2 bytes
typedef shortint = int; // 4 bytes
typedef longint = int;  // 8 bytes
typedef long = int;     // 8 bytes
```

**Comment by simc** [comment]
closing this for now because I haven't seen other users disliking the naming and I don't really see a great alternative
